nextjs/pages/index.js

```diff
@@ -0,0 +1,126 @@
+import Head from "next/head";
+import React, { useCallback, useEffect } from "react";
+import styles from "../styles/RandomGenerator.module.css";
+
+const getColor = (rand) => {
+  //return rand > 0.5 ? "white" : "black";
+  if (rand < 0.33333) {
+    return "deeppink";
+  }
+  if (rand < 0.66666) {
+    return "green";
+  }
+  return "blue";
+};
+
+const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+
+export default function Home() {
+  const [seed, setSeed] = React.useState("default");
+  const [size, setSize] = React.useState(40);
+  const [renderedSize, setRenderedSize] = React.useState(size);
+  const [results, setResults] = React.useState([]);
+  const [loading, setLoading] = React.useState(false);
+
+  const getRandom = async (event, seedArg) => {
+    setLoading(true);
+    if (typeof seedArg === "string") {
+      setSeed(seedArg);
+    }
+    if (event) {
+      event.preventDefault();
+    }
+    const result = await fetch(
+      `/api/random-pixels?seed=${seedArg || seed}&limit=${size * size}`
+    ).then((res) => res.json());
+    setRenderedSize(size);
+    setResults(result.results);
+    setLoading(false);
+    return false;
+  };
+
+  const generateApiLoad = async () => {
+    for (;;) {
+      await getRandom(null, Math.random().toString());
+      await sleep(100);
+    }
+  };
```

Comment:
I looooove that this is so simple to do. No external load testing tool require :sunglasses: